data/operators/telecom/exchange.json

```diff
@@ -242,6 +255,19 @@
         "telecom": "exchange"
       }
     },
+    {
+      "displayName": "MTN Group",
+      "locationSet": {
+        "include": ["za", "ng", "gh", "bj", "bw", "cm", "af", "cg", "gn", "gw", "ir", "ci", "lr", "rw", "ss", "sd", "sy", "sz", "ug", "ye", "zm" ]
```

Comment:
So many countries, I'd prefer think this company is global and use "001" instead.

(MTN's wikidata and wikipedia is correct)